guardian/pressreader
# Update infrastructure setup to create lambda per config

## What does this change?

Configure a lambda per config / write destination in order to run the "new" and "old" paths in parallel and at the same time provide a way to publish different editions.

## How to test

Deploy this to INFRA and ensure that the infrastructure deploys, the lambdas run when triggered via a test run, and that the expected data is available at the appropriate endpoints with the API key retrievable from API gateway:

- https://pressreader.gutools.co.uk/US/20230606
- https://pressreader.gutools.co.uk/AUS/20230606

## How can we measure success?

Pressreader are able to use the data provided by the lambda to produce print on demand papers, and eventually can use the API gateway endpoints without further work.

## Have we considered potential risks?

We need to check that the data the new method produces is functionally substitutable from the old data, and that we will be alerted if there are errors (we can test this by simulating an alert state).
